src/adapter/src/catalog/open.rs

```diff
@@ -1761,6 +1767,24 @@ fn default_logging_config() -> ReplicaLogging {
         interval: Some(Duration::from_secs(1)),
     }
 }
+pub struct BuiltinBootstrapClusterSizes {
+    /// Size to default system_cluster init to
+    pub system_cluster: String,
+    /// Size to default introspection_cluster init to
+    pub introspection_cluster: String,
+}
+
+impl BuiltinBootstrapClusterSizes {
+    fn get_size(&self, cluster_name: &str) -> Result<String, mz_catalog::durable::CatalogError> {
+        match cluster_name {
+            "mz_system" => Ok(self.system_cluster.clone()),
+            "mz_introspection" => Ok(self.introspection_cluster.clone()),
```

Comment:
Can you use the constants here `mz_catalog::builtin::MZ_INTROSPECTION_CLUSTER.name` and `mz_catalog::builtin::MZ_SYSTEM_CLUSTER.name`.

Reply:
Done, although I did have to convert to if/else. This isn't possible with matchers is it?

Reply:
I'm honestly not sure.